# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_redirect_history.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_redirect_history.js
--- a/netwerk/test/unit/test_redirect_history.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_redirect_history.js	Tue Feb 12 16:08:25 2019 +0000
@@ -64,6 +64,6 @@
   var uri = NetUtil.newURI("http://test.com");
   httpChan = chan.QueryInterface(Ci.nsIHttpChannel);
   httpChan.referrer = uri;
-  chan.asyncOpen2(new ChannelListener(finish_test, null));
+  chan.asyncOpen(new ChannelListener(finish_test, null));
   do_test_pending();
 }